moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1311,11 +1311,13 @@ as the new values that correspond to the group */
   /** \brief Update the state after setting a particular link to the input global transform pose.*/
   void updateStateWithLinkAt(const LinkModel* link, const Eigen::Isometry3d& transform, bool backward = false);
 
+  /** \brief Get the transform of the given link with respect to the root link of the URDF. */
```

Comment:
Actually it is the root link of MoveIt's RobotModel, which might differ from the URDF root link if the SRDF added a new virtual joint at the root.